build.gradle.kts

```diff
@@ -111,6 +111,12 @@ configurations {
     compatTestRuntimeClasspath {
         extendsFrom(testRuntimeClasspath.get())
     }
+    create("e2eTestCompileClasspath") {
```

Comment:
Can I generate something in Kotlin to be able to refer to them by name?

Reply:
You shouldn't have to create these configurations since the source set will already create them for you. After creating the source set, you can configure them like this:

```kotlin
configurations.named(e2eTest.implementationConfigurationName) {
    extendsFrom(configurations.testImplementation.get())
}

configurations.named(e2eTest.runtimeOnlyConfigurationName) {
    extendsFrom(configurations.testRuntimeOnly.get())
}
```